client/index.js

```diff
@@ -70,25 +70,19 @@ addFilter(
 			capability: 'manage_woocommerce',
 		} );
 
-		// Currently under feature flag.
-		if (
-			wcpaySettings &&
-			wcpaySettings.featureFlags.isEmbeddedKycEnabled
```

Comment:
nitpick: you should also remove `isEmbeddedKycEnabled` from `globals.d.ts`

Reply:
Done in https://github.com/Automattic/woocommerce-payments/pull/9445/commits/7ca58a8f8f51d5df09e4a187d5cf270f73396667